### ISSUE TITLE
Some emoji's can't be unemojified

### ISSUE THREAD

**Issue by Fintasys** [GatchHQ/dart-emoji#22]
Hey! Thank you for taking up this abandoned library 🙇 

I've noticed that some emoji's can't be unemojified, probably because the emoji consist of multiple elements like the one which are gender specific. 

```
  test('unemojify a text ', () {
    expect(emojiParser.unemojify('🚣‍♂️'), ':man-rowing-boat:');
    expect(emojiParser.unemojify('🏄‍♂️'), ':man-surfing:');
  });
```

I also needed to replace the `rowboat` emoji with this one `🚣` to receive a result like this `:rowboat:‍:male_sign:'`
Seems some of the neutral emoji's are not correct yet.

**Comment by nilsreichardt** [comment]
Hi @Fintasys, thanks for your report! I can reproduce the issue. #21 is also related to this issue. I'll take a look

**Comment by nilsreichardt** [comment]
@Fintasys I fixed it with https://github.com/GatchHQ/dart-emoji/pull/24. I'll make a few more improvements and push a new version in next days to pub.dev 👍

**Comment by nilsreichardt** [comment]
@Fintasys I published `v0.2.0` on pub.dev. Additionally, I added in https://github.com/GatchHQ/dart-emoji/pull/25 +1.750 tests to check if you can unemojifiy an emoji 👍
